server/container/guice/guice-common/src/main/java/org/apache/james/GuiceJamesServer.java

```diff
@@ -80,7 +81,7 @@ public void start() throws Exception {
 
     public void stop() {
         if (preDestroy != null) {
-            cleanupTasksPerformer.clean();
+            Optional.ofNullable(cleanupTasksPerformer).ifPresent(CleanupTasksPerformer::clean);
```

Comment:
I think it should not be in `if (preDestroy != null)` block

Reply:
true. I fixed this.